omod/src/main/webapp/resources/htmlforms/KP_PP_Classification_Tool.xml

```diff
@@ -43,11 +43,280 @@
         form .lablex {
         width: 93%;
         }
-        
+
 
     </style>
+    <script type="text/javascript">
+        if(jQuery){
+          jQuery(document).ready(function(){
+
+                 jq("#msm_final").find(":checkbox").attr("disabled","disabled");
```

Comment:
@balukujr Can you also ensure that the KP classifications, not just the final ones are also disabled when the page loads. For example I can be Male at Birth, and think of myself as Male but someone can select TG.